api/MediaDevices.json

```diff
@@ -112,15 +112,35 @@
             "firefox": [
               {
                 "version_added": "63",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "media.setsinkid.enabled",
+                    "value_to_set": "true"
+                  }
+                ],
+                "notes": "Prior to Firefox 63, <code>enumerateDevices()</code> only returned input devices. Starting with Firefox 63, output devices are also included."
```

Comment:
Given the flag, the `notes` field for both firefox and firefox_android should have the following text added to the end of the sentence: `"if the <code>media.setsinkid.enabled</code> preference is enabled."`